doc/release_notes/release_notes_2.4.rst

```diff
@@ -0,0 +1,87 @@
+.. _release_notes_2.4:
+
+ACRN v2.4 (Mar 2021) - DRAFT
+############################
+
+We are pleased to announce the release of the Project ACRN
+hypervisor version 2.4.
+
+ACRN is a flexible, lightweight reference hypervisor that is built with
+real-time and safety-criticality in mind. It is optimized to streamline
+embedded development through an open source platform. Check out the
+:ref:`introduction` introduction for more information.  All project ACRN
+source code is maintained in the
+https://github.com/projectacrn/acrn-hypervisor repository and includes
+folders for the ACRN hypervisor, the ACRN device model, tools, and
+documentation. You can either download this source code as a zip or
+tar.gz file (see the `ACRN v2.4 GitHub release page
+<https://github.com/projectacrn/acrn-hypervisor/releases/tag/v2.4>`_) or
+use Git clone and checkout commands::
+
+   git clone https://github.com/projectacrn/acrn-hypervisor
+   cd acrn-hypervisor
+   git checkout v2.4
+
+The project's online technical documentation is also tagged to
+correspond with a specific release: generated v2.4 documents can be
+found at https://projectacrn.github.io/2.4/.  Documentation for the
+latest under-development branch is found at
+https://projectacrn.github.io/latest/.
+
+ACRN v2.4 requires Ubuntu 18.04.  Follow the instructions in the
+:ref:`rt_industry_ubuntu_setup` to get started with ACRN.
+
+
+What's New in v2.4
+******************
+
+
+Document Updates
+****************
+
+New and updated reference documents are available, including:
+
+.. rst-class:: rst-columns2
+
+* :ref:`GVT-g_api`
```

Comment:
GVT-g really isn't used much and I think these updates were mostly about deleting i915 driver parameters that became obsolete. Can we put those at the bottom of the list, I do not want people to get the impression the GVT-g stuff is up-to-date.

Reply:
Sure, this list is generated by a script that looks for docs with more than 20 lines changed since the v2.3 release (as computed by ``git diff --stat `git rev-parse v2.3` `git rev-parse master` | grep \.rst``).  After that, we can edit which ones we actually include in the release notes and their order.